packages/mds-jurisdiction/handlers/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { ServerError } from '@mds-core/mds-utils'
+import { Jurisdiction } from '@mds-core/mds-types'
+import { JurisdictionApiResponse } from '../types'
+
+export const UnexpectedServiceError = (error: ServerError | null) =>
```

Comment:
Do you foresee this being used elsewhere? If so, would make sense in `mds-utils/exceptions.ts`

Reply:
Perhaps, yes. I will look for other opportunities to use it and perhaps move it then.